test/index.html

```diff
@@ -0,0 +1,27 @@
+<html>
+
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width">
+    <title>QUnit Example</title>
+    <link rel="stylesheet" href="http://sdk.medkumo.loc/test/lib/qunit-2.0.1.css">
+</head>
+
+<body>
+    <div id="qunit"></div>
+    <div id="qunit-fixture"></div>
+    <script src="http://sdk.medkumo.loc/test/lib/qunit-2.0.1.js"></script>
+    <script>
+        (function() {
+            var script = document.createElement('script');
+            script.async = true;
+            script.src = 'http://sdk.medkumo.loc/medkumo.js';
+            var entry = document.getElementsByTagName('script')[0];
+            entry.parentNode.insertBefore(script, entry);
+        })();
+    </script>
```

Comment:
add this code before </script>


window.Medkumo_ready = function() {
    Medkumo.init('<your hospital key>', 'test');
  };